octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/util/icon/EntityIconFactory.java

```diff
@@ -85,26 +88,66 @@ private void init() {
 
     private void loadImageData(Entity entity) {
         IconDetail iconDetail = iconDetailMap.containsKey(entity) ? iconDetailMap.get(entity) : undefinedIconDetail;
-
+           
         Display display = Display.getDefault();
+        Color white = display.getSystemColor(SWT.COLOR_WHITE);
```

Comment:
Why do you need this variable declared in so many places? Couldn't you declare it in your class and use it wherever you need it, or just use te display.getSystemColor(SWT.COLOR_WHITE)?